package.json

```diff
@@ -16,7 +16,7 @@
     "preemulator": "npm run emulator:boot",
     "emulator": "adb wait-for-device && sleep 5 && npm run emulator:build && npm run emulator:test",
     "postemulator": "npm run emulator:kill",
-    "prerelease": "npm run emulator",
+    "prerelease": "npm ci && npm run emulator",
```

Comment:
Why `npm ci` over `npm install` ?

Reply:
Mainly because it doesn't update the package lock. But also because it removes the node_modules first which I think gives us even more assurance that we a clean and updated environment before the release.

Reply:
Ah interesting, i had no idea it removed `node_modules` first ...